apps/workers/sql-load/worker.go

```diff
@@ -124,33 +125,69 @@ func (w *worker) DoTask(ctx context.Context) (task.Result, string) {
 
 	// read the files for loading, verify columns types
 	start := time.Now()
-	err = w.ReadFiles()
+	err = w.ds.ReadFiles(w.flist, w.fileOpts, w.Params.SkipErr)
 	if err != nil {
-		return task.Failed(errors.Wrap(err, "readfiles error"))
+		return task.Failed(fmt.Errorf("readfiles error %w", err))
```

Comment:
You could use `task.Failf("readfiles error %w", err)`

Reply:
The fmt.Errorf will wrap the given error if you use the %w